src/Compilers/CSharp/Test/Semantic/Semantics/BindingAsyncTests.cs

```diff
@@ -1214,10 +1214,14 @@ interface IInterface
 {
     async void F(); 
 }";
-            CreateCompilationWithMscorlib45(source).VerifyDiagnostics(
-                // (4,16): error CS0106: The modifier 'async' is not valid for this item
+            CreateCompilationWithMscorlib45(source, parseOptions:TestOptions.Regular7).VerifyDiagnostics(
+                // (4,16): error CS8503: The modifier 'async' is not valid for this item in C# 7. Please use language version 7.1 or greater.
```

Comment:
Since the feature is currently targeting 8.0, you may want to create LanguageVersion 8.0 (in master branch) and save going back and updating tests later on.